src/main/webapp/WEB-INF/tags/instructor/results/filterEditModal.tag

```diff
@@ -0,0 +1,116 @@
+                        <%=Const.NO_SPECIFIC_RECIPIENT%>
+                      </option>
+                    </select>
+                  </div>
+                </div>
+              </c:if>
+            </div>
+            <div class="col-md-5">
+              <label class="control-label">
+                Additional settings:
+              </label>
+              <div data-toggle="tooltip" title="Group results in the current view by team">
+                <div class="checkbox">
+                  <label<c:if test="${filterPanel.sortType == 'question'}"> class="text-strike"</c:if>>
+                    <input type="checkbox" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYTEAM%>"
+                           id="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYTEAM%>"
+                           <c:if test="${filterPanel.groupedByTeam}">checked</c:if>
+                           <c:if test="${filterPanel.sortType != 'question'}">class="checkbox-group-by-team"</c:if>> Group by Teams
+                  </label>
+                </div>
+              </div>
+              <div data-toggle="tooltip" title="Show statistics">
+                <div class="checkbox">
+                  <label<c:if test="${filterPanel.sortType == 'recipient-giver-question' or filterPanel.sortType == 'giver-recipient-question'}"> class="text-strike"</c:if>>
+                    <input type="checkbox" id="show-stats-checkbox"
+                           name="<%=Const.ParamsNames.FEEDBACK_RESULTS_SHOWSTATS%>"
+                    <c:if test="${filterPanel.statsShown}"> checked</c:if>> Show Statistics
+                  </label>
+                </div>
+              </div>
+              <div data-toggle="tooltip" title="Indicate missing responses">
+                <div class="checkbox">
+                  <input type="checkbox"
+                         id="indicate-missing-responses-checkbox"
+                         value="true"
+                         name="<%=Const.ParamsNames.FEEDBACK_RESULTS_INDICATE_MISSING_RESPONSES%>"
+                  <c:if test="${filterPanel.missingResponsesShown}"> checked</c:if>>
+                  Indicate Missing Responses
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_SESSION_NAME%>"
+                 value="${filterPanel.feedbackSessionName}">
+          <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>"
+                 value="${filterPanel.courseId}">
+          <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>"
+                 value="${data.account.googleId}">
+
+          <div class="modal-footer">
+            <button id="submitBtn" type="submit" class="btn btn-primary pull-left">Change View</button>
+            <button type="button" class="btn btn-danger pull-right" data-dismiss="modal">Cancel</button>
+          </div>
```

Comment:
Agree; the cancel button should be uncoloured for consistency with our other modals:
<img width="612" alt="screen shot 2017-07-27 at 7 45 42 pm" src="https://user-images.githubusercontent.com/6811428/28668820-3de3842c-7304-11e7-830b-4a99ec651b6a.png">